### PR TITLE
doc(php): add procedure to upgrade to php 7.3 on 20.04.x

### PR DESCRIPTION
**Pull request by kduret**
## Description

add procedure to upgrade php on 20.04.x

## Target serie

- [x] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Review by jeremyjaouen — APPROVED** [review]

File: fr/administration/secure-platform.md [hunk 1]
```diff
@@ -237,6 +237,33 @@ Pour mettre à jour l'URI Centreon, vous devez suivre les étapes suivantes:
...
+
+PHP 7.2 n'est plus supporté depuis novembre 2020
+
+Si vous ne pouvez pas mettre à jour votre plateforme vers la prochaine version majeure de Centreon, et que vous souhaitez obtenir les derniers correctifs de sécurité, vous devez suivre les étapes suivantes:
+
+1. Vérifier les versions requises de centreon web & des extensions:
+    * centreon-web >= 20.04.17
+    * centreon-bi >= 20.04.8
+    * centreon-open-tickets >= 20.04.3
+
```
Comment (cg-tw) [inline fr/administration/secure-platform.md:250]: ajouter qu'il doit faire `rpm -qa | grep centreon` pour connaître la version de ses composants

File: en/administration/secure-platform.md [hunk 2]
```diff
@@ -374,6 +374,33 @@ To update the Centreon URI, you need to follow those steps:
...
+
+PHP 7.2 has reached end of life since november of 2020
+
+If you cannot upgrade your platform to the next major version of Centreon, and want to get last security fixes, you need to follow those steps:
+
+1. Check centreon web & extensions version requirements:
+    * centreon-web >= 20.04.17
+    * centreon-bi >= 20.04.8
+    * centreon-open-tickets >= 20.04.3
+
```
Comment (cg-tw) [inline en/administration/secure-platform.md:387]: Add that they must do `rpm -qa | grep centreon` to know their versions

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: en/administration/secure-platform.md [hunk 3]
```diff
@@ -374,6 +374,37 @@ To update the Centreon URI, you need to follow those steps:
...
 change **/centreon** path with your new path
 
+## Upgrade to PHP 7.3
+
+PHP 7.2 reached its end of life in November 2020.
+
+If you cannot upgrade your platform to the next major version of Centreon, and want to get the last PHP security fixes, you need to follow these steps:
+
+1. Check centreon web & extensions version requirements:
+    * centreon-web >= 20.04.17
```
Comment (tuntoja) [inline en/administration/secure-platform.md:384]:
```suggestion
    * centreon-web >= 20.04.18
```

File: fr/administration/secure-platform.md [hunk 4]
```diff
@@ -237,6 +237,37 @@ Pour mettre à jour l'URI Centreon, vous devez suivre les étapes suivantes:
...
 et modifiez le chemin **/centreon** par le nouveau.
 
+## Mise à jour vers PHP 7.3
+
+PHP 7.2 n'est plus supporté depuis novembre 2020.
+
+Si vous ne pouvez pas mettre à jour votre plateforme vers la prochaine version majeure de Centreon, et que vous souhaitez obtenir les derniers correctifs de sécurité de PHP, vous devez suivre les étapes suivantes:
+
+1. Vérifier les versions requises de centreon web & des extensions:
+    * centreon-web >= 20.04.17
```
Comment (tuntoja) [inline fr/administration/secure-platform.md:247]:
```suggestion
    * centreon-web >= 20.04.18
```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Review by adr-mo — APPROVED** [review]

File: en/administration/secure-platform.md [hunk 5]
```diff
@@ -374,6 +374,37 @@ To update the Centreon URI, you need to follow those steps:
...
 change **/centreon** path with your new path
 
+## Upgrade to PHP 7.3
+
+PHP 7.2 reached its end of life in November 2020.
+
+If you cannot upgrade your platform to the next major version of Centreon, and want to get the last PHP security fixes, you need to follow these steps:
+
+1. Check centreon web & extensions version requirements:
+    * centreon-web >= 20.04.18
```
Comment (thiuyendang) [inline en/administration/secure-platform.md:384]:
```suggestion
    * centreon-web >= 20.04.19
```

File: fr/administration/secure-platform.md [hunk 6]
```diff
@@ -237,6 +237,37 @@ Pour mettre à jour l'URI Centreon, vous devez suivre les étapes suivantes:
...
 et modifiez le chemin **/centreon** par le nouveau.
 
+## Mise à jour vers PHP 7.3
+
+PHP 7.2 n'est plus supporté depuis novembre 2020.
+
+Si vous ne pouvez pas mettre à jour votre plateforme vers la prochaine version majeure de Centreon, et que vous souhaitez obtenir les derniers correctifs de sécurité de PHP, vous devez suivre les étapes suivantes:
+
+1. Vérifier les versions requises de centreon web & des extensions:
+    * centreon-web >= 20.04.18
```
Comment (thiuyendang) [inline fr/administration/secure-platform.md:247]:
```suggestion
    * centreon-web >= 20.04.19
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Comment by kduret** [comment]
mbi prequisite version will not be released on 20.04

**Pull request closed without merging**